gradle.properties

```diff
@@ -1,4 +1,4 @@
 group=com.vestmark.nexus.plugins
 version=1.5.0.0
```

Comment:
Before you go too much further, you will want to update this version too, as we are releasing a new version of our own plugin.  Jump to 1.6.0.0 please.  Thanks!